### PR TITLE
rewrite of where Appraisal policies originate from

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
...
 This procedure is called the appraisal of Evidence.
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner,
+and/or is configured into the Verifier by the Verifier Owner,
+and/or is programmed into the Verifier,
+and/or might be obtained via some other mechanism.
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md:382]: So, if I choose "and" at every decision-point, is that statement still true?

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
...
 A Relying Party uses Attestation Results by applying its own
 appraisal policy to make application-specific decisions, such as authorization decisions.
-The Appraisal Policy for Attestation Results is configured in the Relying Party by the Relying Party Owner,
-and/or are programmed into the Relying Party.
 This procedure is called the appraisal of Attestation Results.
 
+The Appraisal Policy for Attestation Results might be obtained from the Relying Party Owner,
+and/or is configured into the Relying Party by the Relying Party Owner,
+and/or is programmed into the Relying Party,
+and/or might be obtained via some other mechanism.
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md:391]: Same as above

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester.
 This procedure is called the appraisal of Evidence.
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner,
+and/or is configured into the Verifier by the Verifier Owner,
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:380]: "configured into" is one way of obtaining it, so 379-380 are redundant

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (dthaler) [reply draft-ietf-rats-architecture.md:382]: I think just "or" would be better, to Henk's point.

File: draft-ietf-rats-architecture.md (hunk 3 above)
Comment (mcr) [inline draft-ietf-rats-architecture.md:380]:
```suggestion
The Appraisal Policy for Evidence might be obtained from the Verifier Owner via some protocol mechanism
and/or is configured into the Verifier by the Verifier Owner,
```

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (mcr) [inline draft-ietf-rats-architecture.md:382]:
```suggestion
or is configured into the Verifier by the Verifier Owner,
or is programmed into the Verifier,
or might be obtained via some other mechanism.
```

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester.
 This procedure is called the appraisal of Evidence.
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner via some protocol mechanism
+or is configured into the Verifier by the Verifier Owner,
+or is programmed into the Verifier,
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:381]:
```suggestion
or might be configured into the Verifier by the Verifier Owner,
or might be programmed into the Verifier,
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
...
 A Relying Party uses Attestation Results by applying its own
 appraisal policy to make application-specific decisions, such as authorization decisions.
-The Appraisal Policy for Attestation Results is configured in the Relying Party by the Relying Party Owner,
-and/or are programmed into the Relying Party.
 This procedure is called the appraisal of Attestation Results.
 
+The Appraisal Policy for Attestation Results might be obtained from the Relying Party Owner,
+and/or is configured into the Relying Party by the Relying Party Owner,
+and/or is programmed into the Relying Party,
+and/or might be obtained via some other mechanism.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:391]:
```suggestion
The Appraisal Policy for Attestation Results might be obtained from the Relying Party Owner via some protocol mechanism,
or might be configured into the Relying Party by the Relying Party Owner,
or might be programmed into the Relying Party,
or might be obtained via some other mechanism.
```

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -375,16 +375,20 @@ A Verifier uses the Evidence, any Reference Values from Reference Value Provider
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester.
 This procedure is called the appraisal of Evidence.
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner via some protocol mechanism
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:379]:
```suggestion
The Appraisal Policy for Evidence might be obtained from the Verifier Owner via some protocol mechanism,
```

**Pull request merged**